### DISCUSSION TITLE
如何实现条件判断？

### DISCUSSION THREAD

**Discussion by pooneyy** [qd-today/qd#492] [Q&A]
我在文档中并未找到相关的说明

**Comment by a76yyyy** [comment]
框架首页左上角 "常用API/过滤器"-"常用参数及过滤器"

**Reply by pooneyy** [reply]
![image](https://github.com/qd-today/qd/assets/85266337/ade3756a-89e0-4005-aea8-70976f14539a)
问题是没写具体的用法呀哥哥
![image](https://github.com/qd-today/qd/assets/85266337/9cd285ed-801b-44f6-9684-9c06b569b040)
 不知道怎么写

**Reply by a76yyyy** [reply]
参考模板https://github.com/qd-today/templates/blob/b442b8b20a540496453693ae1e3d73bc8c4b2fe5/%E9%9B%A8%E4%BA%91%E7%AD%BE%E5%88%B0.har#L53

**Reply by pooneyy** [reply]
你好，首先谢谢你的模板举例，我还有一个疑问，有没有类似 else if 的语法呢？如果碰到多个条件，是使用多个{% if 表达式 %}吗？
![image](https://github.com/qd-today/qd/assets/85266337/cfd5e9c2-b515-4bcd-bd41-bad1062eca1a)
如果遇到需要嵌套的if，又如何处理呢？

**Reply by a76yyyy** [reply]
支持if else的嵌套

**Reply by pooneyy** [reply]
你好，我还有一个问题，如何更新别人发布的模板呢？

**Reply by a76yyyy** [reply]
前往 [模板库](https://github.com/qd-today/templates) 指定模板的评论区回复更新内容并@模板作者或者仓库管理员即可
